code/datums/traits/negative.dm

```diff
@@ -328,6 +328,23 @@
 		if(I.fingerprintslast == quirk_holder.ckey)
 			quirk_holder.put_in_hands(I)
 
+/datum/quirk/phobia
+	name = "Phobia"
+	desc = "You have a debilitating fear you just can't seem to overcome."
+	value = -2
+	gain_text = null
+	lose_text = null
+	medical_record_text = "Patient has a deep rooted phobia."
+
+/datum/quirk/phobia/add()
+	var/datum/brain_trauma/mild/phobia/T = new()
+	var/mob/living/carbon/human/H = quirk_holder
+	H.gain_trauma(T, TRAUMA_RESILIENCE_ABSOLUTE)
+
+/datum/quirk/phobia/remove()
+	var/mob/living/carbon/human/H = quirk_holder
+	H.cure_trauma_type(resilience = TRAUMA_RESILIENCE_ABSOLUTE)
```

Comment:
pretty sure this can just be `qdel(the_phobia)`, I don't see a "cure (specific) trauma" type proc.

Reply:
unless i did something wrong qdel does not appear to work, but im not sure if im defining the phobia properly

Reply:
VV the quirk and see if it refers to the phobia on the mob?